.readthedocs.yml

```diff
@@ -0,0 +1,4 @@
+version: 2
+
```

Comment:
I think as per [this paragraph](https://docs.readthedocs.io/en/stable/config-file/v2.html#sphinx)  we also need to specify where our `conf.py` file is located.

Reply:
Sir I have specified the location of conf.py.